pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -785,7 +785,10 @@ type GCESDConfig struct {
 // +k8s:openapi-gen=true
 type OpenStackSDConfig struct {
 	// The OpenStack role of entities that should be discovered.
-	// +kubebuilder:validation:Enum=Instance;instance;Hypervisor;hypervisor
+	//
+	// Note: The `loadbalancer` or `Loadbalancer` role requires Prometheus >= v3.2.0.
+	//
+	// +kubebuilder:validation:Enum=Instance;instance;Hypervisor;hypervisor;Loadbalancer;loadbalancer
```

Comment:
(nit)
```suggestion
	// +kubebuilder:validation:Enum=Instance;Hypervisor;Loadbalancer
```

Reply:
(suggestion) It might make sense here to define a `OpenStackRole` type. Something like:
```
// +kubebuilder:validation:Enum=Instance;Hypervisor;Loadbalancer
type OpenStackRole string

const (
	OpenStackRoleInstance     OpenStackRole = "Instance"
	OpenStackRoleHypervisor   OpenStackRole = "Hypervisor"
	OpenStackRoleLoadBalancer OpenStackRole = "LoadBalancer"
)
```

We will also need to update the rest of the code base to use these constants.

Reply:
we're ok breaking the API here because it's v1alpha1. In fact we plan on making all breaking changes before v1beta1. I would only advise to remove the lower-case enums in a separate PR. For this PR, I'd add only `LoadBalancer`.

Reply:
Thanks @simonpasquier @mviswanathsai 
Add only `LoadBalancer` in this PR. 
Will remove  the lower-case enums in next PR.